pseudopotential_calculator/calculations/bulk.py

```diff
@@ -155,44 +209,37 @@ def plot_energy_against_n_k_points(
     )
 
 
-def _get_cutoff_energy_from_calculator(
-    calculator: Castep,
-) -> float:
-    return cast(
-        float,
-        calculator.param.cut_off_energy.raw_value[0],  # type: ignore unknown
-    )
-
-
-def _plot_energy_against_cutoff_energy(
-    cutoff_energy: np.ndarray[Any, np.dtype[np.float64]],
-    energy: np.ndarray[Any, np.dtype[np.float64]],
+def plot_energy_against_cutoff_energy(
+    calculators: list[Castep],
     *,
     ax: Axes | None = None,
 ) -> tuple[Figure, Axes, Line2D]:
-    fig, ax = get_figure(ax)
+    cutoff_energy = list[float]()
+    energies = list[float]()
+    for calculator in calculators:
+        energies.append(cast(Atoms, calculator.atoms).get_potential_energy())  # type: ignore inkown
```

Comment:
Add a get_atoms function to catep.py, and handle the case when atoms is none

Reply:
```   
for calculator in calculators:
        energies.append(cast(Atoms, get_calculator_atom(calculator)).get_potential_energy())  # type: ignore inkown
        cutoff_energy.append(_get_cutoff_energy_from_calculator(calculator))
```

```
def get_calculator_atom(calculator: Castep) -> Atoms:
    return calculator.atoms  # type: ignore this is the only way # noqa: SLF001
```


is it like this? why do we do it? for consistency?